rotkehlchen/chain/ethereum/balancer/typing.py

```diff
@@ -0,0 +1,74 @@
+from typing import (
+    Any,
+    DefaultDict,
+    Dict,
+    List,
+    NamedTuple, NewType,
+    Optional,
+    Set,
+)
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.constants import ZERO
+from rotkehlchen.fval import FVal
+from rotkehlchen.typing import ChecksumEthAddress
+
+
+# Below set of classes where to store PoolShare, Pool and PoolToken data
+# BalancerPoolAsset stores PoolToken data, and adds custom ones
+class BalancerPoolAsset(NamedTuple):
+    address: ChecksumEthAddress  # PoolToken.id, token contract address
+    balance: FVal  # PoolToken.balance
+    denorm_weight: FVal  # PoolToken.denormWeight
+    name: str  # PoolToken.name
+    symbol: str  # PoolToken.symbol
+    # Custom fields
+    user_balance: FVal  # Estimated token balance
+    user_balance_usd: Optional[FVal] = ZERO  # Estimated token balance in USD
+    asset: Optional[Asset] = None
+    asset_usd: Optional[FVal] = ZERO  # price in USD per token
```

Comment:
Since this is Price perhaps use the `Price` type?

Reply:
I'll do

Reply:
done